src/Redux/api.tsx

```diff
@@ -130,6 +130,10 @@ const routes: Routes = {
     path: "/api/v1/getallfacilities",
   },
 
+  getAllFacilities: {
+    path: "/api/v1/getallfacilities/{id}/",
+  },
+
```

Comment:
Can you rename it to like `getFacilityV2` and place it below old api so that its easier to track which endpoint we used and using.